manifests/storage_state_crd.yaml-patch

```diff
@@ -0,0 +1,6 @@
+- op: add
+  path: /spec/versions/name=v1alpha1/schema/openAPIV3Schema/properties/metadata/properties
```

Comment:
what does this do?

Reply:
readd the metadata.name schema. It looks like it was just manually added to the CRD manifest and controller-tools will override that.